_ext/posts.rb

```diff
@@ -0,0 +1,137 @@
+require 'nokogiri'
+
+module Awestruct
+  module Extensions
```

Comment:
Is this extension coming from somewhere else btw? Or is it written all from scratch?

Reply:
It looks like its very similar to the one for in.relation.to.  I did make a few modifications to the teaser role below which I don't believe their version supports.